Automattic/wp-calypso
# Odyssey Stats: Add a Jetpack upsell section to the traffic page

Fixes #71820.

Design|Implementation
-|-
![Image](https://user-images.githubusercontent.com/6406071/213558155-f7dad638-85d7-471d-9f40-a49621bdd8bf.png)|<img width="1254" alt="image" src="https://user-images.githubusercontent.com/4044428/221996392-79ddf5f1-2ed1-4212-bd3f-c277725fe0bf.png">
<img width="233" alt="image" src="https://user-images.githubusercontent.com/4044428/222274728-407706a2-fbde-4758-9b33-3de7630351d7.png">|<img width="446" alt="image" src="https://user-images.githubusercontent.com/4044428/222275691-ef778174-bbe7-48a9-a904-2713b726926a.png">

## Proposed Changes

Integrates a new Jetpack Upsell Card into a new section on the Odyssey Stats Traffic page. Available upsells are determined by excluding currently active products for the Jetpack site; the corresponding CTA buttons link to the WordPress.com checkout pages.

## Testing Instructions



* Apply this change to your Odyssey Stats instance using option 1 or 2 from PejTkB-3E-p2.
* Open Odyssey Stats and navigate to the bottom of the traffic tab.
* Ensure you see the new Jetpack upsell section.
* Spin up the component package storybook and ensure the Jetpack Upsell Card stories work as expected.
(`yarn workspace @automattic/components run storybook`)

## Pre-merge Checklist
